corehq/apps/domain/forms.py

```diff
@@ -2129,6 +2131,25 @@ def save(self, commit=True):
             )
             return False
 
+    def send_prepayment_invoice(self, date_start, date_end):
+        email_list = self.cleaned_data['email_list']
+        contact_emails = [email_list[0]]
+        cc_emails = [email for email in email_list[1:]]
+        invoice_factory = DomainWireInvoiceFactory(
+            self.domain, date_start=date_start, date_end=date_end,
+            contact_emails=contact_emails, cc_emails=cc_emails
+        )
+        date_due = max(date_start,
+                       datetime.date.today() + datetime.timedelta(days=SUBSCRIPTION_PREPAY_MIN_DAYS_UNTIL_DUE))
+        label = f"One month of {self.plan_version.plan.name}"
+        monthly_fee = self.plan_version.product_rate.monthly_fee
+        duration = relativedelta(date_end, date_start)
+        num_months = duration.years * 12 + duration.months
+        amount = monthly_fee * num_months
+        invoice_factory.create_wire_credits_invoice(
+            amount, label, monthly_fee, num_months, date_due
+        )
```

Comment:
For my own understanding, what exactly do we mean by "wire credits" in this name?